frontend/src/components/CardDetailsPage.tsx

```
@@ -55,12 +67,45 @@ const DetailsCard = ({
     <div className="min-h-screen bg-white p-8 text-gray-600 dark:bg-[#212529] dark:text-gray-300">
       <div className="mx-auto max-w-6xl">
         <div className="mt-4 flex flex-row items-center">
           <div className="flex w-full items-center justify-between">
             <h1 className="text-4xl font-bold">{title}</h1>
+            {type === 'program' &&
+              admins?.some(
+                (admin) => admin.login === ((data as ExtendedSession)?.user?.login as string)
+              ) && (
+                <div className="flex gap-2">
+                  <ActionButton
+                    children=" Add Module"
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)